tracing-core/src/dispatcher.rs

```diff
@@ -305,6 +305,7 @@ impl error::Error for SetGlobalDefaultError {}
 ///
 /// [dispatcher]: ../dispatcher/struct.Dispatch.html
 #[cfg(feature = "std")]
+#[cfg_attr(docsrs, doc(cfg(feature = "std")))]
```

Comment:
this isn't necessary. the `get_default` function is _present_ regardless of the feature flag, it just has a different implementation